assets/src/edit-story/components/canvas/multiSelectionMovable.js

```diff
@@ -137,6 +145,35 @@ function MultiSelectionMovable( { selectedElements, nodesById } ) {
 			onRotateGroupEnd={ ( { targets } ) => {
 				onGroupEventEnd( { targets, isRotate: true } );
 			} }
+
+			onResizeGroupStart={ ( { events } ) => {
+				events.forEach( ( ev, i ) => {
+					const frame = frames[ i ];
+					ev.setOrigin( [ '%', '%' ] );
+					if ( ev.dragStart ) {
+						ev.dragStart.set( frame.translate );
+					}
+				} );
+			} }
+			onResizeGroup={ ( { events } ) => {
+				events.forEach( ( { target, width, height, drag }, i ) => {
+					const sFrame = frames[ i ];
+					const isText = 'text' === targetList[ i ].type;
+					target.style.width = `${ width }px`;
+					target.style.height = `${ height }px`;
+					if ( isText ) {
+						// For text: update font size, too.
+						target.style.fontSize = calculateFitTextFontSize( target.firstChild, height, width );
```

Comment:
Shouldn't this also be called in `onGroupEventEnd` to properly update the element? Just like `SingleSelectionMovable` does in `onResizeEnd`

Reply:
Thanks for catching that, fixed in 982719e.